dj/movies/views.py

```diff
@@ -95,23 +94,57 @@ def search_movie(req: django.http.HttpRequest):
     #     return django.http.HttpResponse("Wrong Method GET", status=405)
 
     body = json.loads(req.body.decode("utf-8"))
-    target_movie_id = None
+
     target_movie_name = body['movie']
-    # print(f"you are searching for this movie (lowercase) -> {target_movie_name.lower()}")
 
-    all_movies = Movie.objects.all()
-    for movie in all_movies:
-        # print(movie.title.lower())
+    return django.http.JsonResponse({"data": {"url": "/search/?search_movie=" + str(target_movie_name)}}, status=200)
+
+
+@csrf_exempt
+def search_page(req: django.http.HttpRequest):
+
+
+    target_movie_name = req.GET.get("search_movie", "default")
+
+    search_for = req.GET.get("movie", None)
+    # print(f"searching for {search_for}")
+
+    searched_movie_list = Movie.objects.filter(title__startswith=search_for).order_by("-updated_on")
+
+
+    list_of_searched_movies = []
+    for movie in searched_movie_list:
+        rating = give_rating(movie.id)
+        item = {
+            "title": movie.title,
+            "url": "http://127.0.0.1:8000/movie/?id=" + str(movie.id),
```

Comment:
Why do we need to pass the host address, `/movie/?id=` should work.

Reply:
yes it's working.